packages/table/src/table.tsx

```diff
@@ -530,14 +561,34 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     }
 
     private bodyCellRenderer = (rowIndex: number, columnIndex: number) => {
-        return this.getColumnProps(columnIndex).renderCell(rowIndex, columnIndex);
+        const columnProps = this.getColumnProps(columnIndex);
+        const cell = columnProps.renderCell(rowIndex, columnIndex);
+        const { loading: cellLoading } = cell.props;
```

Comment:
will do. and yup, it's pretty convenient when implementing `shouldComponentUpdate`